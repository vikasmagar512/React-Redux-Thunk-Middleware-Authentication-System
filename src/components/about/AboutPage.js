
import React,{Component} from 'react';
import createReactClass from  'create-react-class';


import PropTypes from 'prop-types'
import { Link, IndexLink } from 'react-router';

class AboutPage extends Component{
  render() {
    return (
      <div>
        <p>Explore below section to know more about your team and also let other's know about you.</p>
          <ul className="nav nav-tabs" style={{marginBottom:'30px'}}>
            <li><Link to="/about/profile" className="active"> Profile  </Link></li>
            <li><Link to="/about/team" > Team</Link></li>
            <li><Link to="/about/contact" > Contact</Link></li>
          </ul>
        {this.props.children}
      </div>
    );
  }
}
export default AboutPage;
