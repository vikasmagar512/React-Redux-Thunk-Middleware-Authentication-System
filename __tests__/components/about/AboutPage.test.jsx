import React,{Component} from 'react';
import createReactClass from  'create-react-class';
import PropTypes from 'prop-types'
import { Link, IndexLink } from 'react-router';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import AboutPage from '../../../src/components/about/AboutPage';


describe('AboutPage', () => {
  it('should match its empty snapshot', () => {
    const tree = renderer.create(
      <AboutPage/>
     ).toJSON();
    expect(tree).toMatchSnapshot();
  });
/*    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<App/>, div);
    });
*/   
}); 