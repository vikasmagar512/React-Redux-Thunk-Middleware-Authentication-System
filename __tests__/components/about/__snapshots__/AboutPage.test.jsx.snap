// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage should match its empty snapshot 1`] = `
<div>
  <p>
    Explore below section to know more about your team and also let other's know about you.
  </p>
  <p>
    coming soon!
  </p>
  <ul
    className="nav nav-tabs"
  >
    <li>
      <a
        onClick={[Function]}
        style={Object {}}
      >
         Profile  
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}
      >
         Team
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}
      >
         Contact
      </a>
    </li>
  </ul>
</div>
`;
